platforms/core-runtime/base-services/src/main/java/org/gradle/internal/work/WorkerLimits.java

```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 
-package org.gradle.internal.concurrent;
```

Comment:
💭  This and `DefaultWorkerLimits` were the last to remain in `org.gradle.internal.concurrent` in `:base-services`, so I moved them out, closer to their clients.